### PR TITLE
create quickstart page

### PR DESCRIPTION
**Pull request by xand6r**

### CODE REVIEW

**Comment by xand6r** [comment]
i need the upgrade on the logstore network to be completed so i can finish recording the videos, so those are pending for now.

File: docs/network/quickstart/qiuck-start.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    (message) => {
+      // Do something with the messages as they are received
+      console.log(JSON.stringify(message));
+    }
+  );
+```
+
+### **Demo**
+
+Coming soon!z
```
Comment (rsoury) [inline docs/network/quickstart/qiuck-start.md]: What is this?

File: docs/network/quickstart/qiuck-start.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
+---
+title: 'Quick Start'
+sidebar_position: 1
+---
+
+The logstore network can be thought of as a decentralised time-indexed database, meaning essentially that it is a database which can be queried by time. There are several key actions that must be taken in order to be able to store data to the network and retrieve data from the network.
```
Comment (rsoury) [inline docs/network/quickstart/qiuck-start.md]: Let's run this quick start through ChatGPT for punctuation, or at least refine it manually.
ie. "Log Store Network", not "logstore network".

File: docs/network/quickstart/qiuck-start.md (hunk 1 above)
Comment (xand6r) [reply docs/network/quickstart/qiuck-start.md]: i was going to include some video demos,  but the network was not going through coupled with the upgrade of the network so i just put that there to come back to it. ot i should take it out completely?

File: docs/network/quickstart/qiuck-start.md (hunk 2 above)
Comment (rsoury) [reply docs/network/quickstart/qiuck-start.md]: Also, please fix the file name...

**Pull request merged**